metrics/report/report_dockerfile/dut-details.R

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env Rscript
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+suppressMessages(suppressWarnings(library(tidyr))) # for gather()
+library(tibble)
+library(gridExtra)	# So we can plot multiple graphs together
+suppressMessages(suppressWarnings(library(ggpubr)))		# for ggtexttable
+suppressMessages(library(jsonlite))		# to load the data
+
+resultsfiles=c(
+	"boot-times.json",
+	"footprint-busybox-ksm.json",
+	"footprint-elasticsearch-ksm.json",
+	"footprint-mysql-ksm.json",
+	"memory-footprint.json",
+	"memory-footprint-ksm.json"
+	)
+
+data=c()
+stats=c()
+stats_names=c()
+
+# For each set of results
+for (currentdir in resultdirs) {
+	count=1
+	dirstats=c()
+	for (resultsfile in resultsfiles) {
+		fname=paste(inputdir, currentdir, resultsfile, sep="/")
+		if ( !file.exists(fname)) {
+			#warning(paste("Skipping non-existent file: ", fname))
+			next
+		}
+
+		# Derive the name from the test result dirname
+		datasetname=basename(currentdir)
+
+		# Import the data
+		fdata=fromJSON(fname)
+
+		if (length(fdata$'kata-env') != 0 ) {
+			# We have kata-runtime data
+			dirstats=tibble("Run Ver"=as.character(fdata$'kata-env'$Runtime$Version$Semver))
+			dirstats=cbind(dirstats, "Run SHA"=as.character(fdata$'kata-env'$Runtime$Version$Commit))
+
+			pver=as.character(fdata$'kata-env'$Proxy$Version)
+			pver=sub("^[[:alpha:][:blank:]-]*", "", pver)
+			# uncomment if you want to drop the commit sha as well
+			#pver=sub("([[:digit:].]*).*", "\\1", pver)
+			dirstats=cbind(dirstats, "Proxy Ver"=pver)
+
+			# Trim the shim string
+			sver=as.character(fdata$'kata-env'$Shim$Version)
+			sver=sub("^[[:alpha:][:blank:]-]*", "", sver)
+			# uncomment if you want to drop the commit sha as well
+			#sver=sub("([[:digit:].]*).*", "\\1", sver)
+			dirstats=cbind(dirstats, "Shim Ver"=sver)
+
+			# We could add the shim commit in here...
```

Comment:
Convert into an issue?

Reply:
it's actually a dangling comment - the SHA got handled in the above shim version block (combined with the version number).